.github/workflows/preview.yml

```diff
@@ -1,45 +1,78 @@
 name: Preview
 on:
   pull_request:
-    branches: 
+    branches:
       - master
       - develop
-      - 'release/**'
+      - "release/**"
```

Comment:
Why did we change this? Workflow docs encourage strings be wrapped in [single quotes](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals), as the YML parser correctly handles the input.

Reply:
My vscode setup did some autoformatting stuff. Will fix.